gserver/test/steps.handler.spec.ts

```diff
@@ -70,14 +70,15 @@ describe('geStepDefinitionMatch', () => {
     describe('non-standard strings', () => {
         const nonStandardStrings = [
             [`Given(/I do "aa" something/);`, `I do "aa" something`],
-            [`When('I do \' something');`, `I do \' something`],
-            [`    When('I do something');`, `I do something`],
-            [`"Given(/^Me and "([^"]*)"$/, function ()"`, `^Me and "([^"]*)"$`]
+            [String.raw`When('I do \' something');`, String.raw`I do \' something`], //String.raw needed to ensure escaped values can be read.
+            [`"Given(/^Me and "([^"]*)"$/, function ()"`, `^Me and "([^"]*)"$`],
+            [`Given('the test cookie is set', () => cy.setCookie('TEST_COOKIE', 'true'));`, `the test cookie is set`]
```

Comment:
Sorry, didn't mean to remove an existing test case. :) I'll it back, and ensure it still works. :)

> I did change the format to `string.raw` because the original format with backtick didn't escape the character correctly. I did verify that the code works with real parsing, but the only way to reproduce the testing here was to use string.raw.